locales-pending/premium.ftl

```diff
@@ -149,6 +143,10 @@ exposure-card-label-exposure-type = Exposure type
 exposure-card-label-date-found = Date found
 # Status of the exposure card, could be In Progress, Fixed or Action Needed
 exposure-card-label-status = Status
+# Variables: 
+# $category_label is the data breach exposure type that was leaked. Eg. Email, IP Address.
+# $count is the number of times that the data type was leaked.
+exposure-card-label-and-count = { $category_label }: { $count }
```

Comment:
Are we using the localized category name as is? If so, we have a capitalization problem that I raised before.

Not blocking for this bug, but something to keep in mind as a follow-up. Those categories were created to be used in a list, not as stand-alone items, and that's a problem that we'll keep having and should be approached consistently.